components/Details.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
```

Comment:
Is this extra `<div>` needed? it feels out of place.

Reply:
Not anymore! I leave it behind once I turn decorations into a component, on it!